demo/src/main/java/com/demo/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.demo.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
do not use @Data in Entity classes please

Reply:
replaced with @Getter & @Setter